doc/integrator/api.rst

```diff
@@ -6,4 +6,18 @@ JavaScript API
 Any c2cgeoportal application exposes a JavaScript API. This API can be used in third-party applications,
 like CMS applications.
 
-.. TODO: fill for new API
+In the folder ``geoportal/<package>_geoportal/static-ngeo/api/apihelp/`` we have all the files related to
+the API help.
+
+In the ``geoportal/demo_geoportal/static-ngeo/api/api.css`` file you have the API CSS.
+
+In the ``geoportal/demo_geoportal/static-ngeo/api/index.js`` wou can customise you API with:
+
+.. code:: JavaScript
+
+   constants.themesUrl: '<the theme URL>';
+   constants.projection: <the projection>;
+   constants.supportedProjections: [<the projection>, <an other projection>];
```

Comment:
<another projection>

Reply:
Empty comment?

Reply:
Hmm. I wanted to comment:
another projection
(instead of an other projection)

Reply:
+1 thanks :-)